data/darktableconfig.xml.in

```diff
@@ -23,11 +23,14 @@
         <option>on startup</option>
         <option>on close</option>
         <option>on both</option>
+        <option>on startup (don't ask)</option>
+        <option>on close (don't ask)</option>
+        <option>on both (don't ask)</option>
       </enum>
     </type>
     <default>on close</default>
-    <shortdescription>when to check for database maintenance</shortdescription>
-    <longdescription>this will set when db size is checked for possible maintenance condition and ask user to confirm</longdescription>
+    <shortdescription>check for database maintenance</shortdescription>
+    <longdescription>this will set when db size is checked for possible maintenance condition and whether to ask user to confirm</longdescription>
```

Comment:
I'm no native English but I have difficulties to parse this. I would say:

> this option indicates when db size is ...

Reply:
"Sets threshold for database maintenance, with user to authorise" ??

Reply:
Arght. yeah. This is bull****. It's wrong both grammatically and factually. For best ux we need a message that's true and doesn't hide anything, while at the same time needs to assume the person setting this up doesn't know anything about `database`. My scenario assumes a person that sets this is a photographer using darktable because reasons (eg. Adobe hiked the price enough that a DSLR/MILC owner cannot justify it and heard about dt being decent alternative), sees the maintenance message (which mentions that it's possible to change maintenance settings) and needs to make informed choice.

I don't want to pollute commits so my proposition:
`maintenance_check`:
shortdesc: "check for database maintenance"
longdesc: "when to check and ask to perform database maintenance. the don't aks options will simply perform maintenance if needed."

`maintenance_freepage_ratio`:
shortdesc: "database fragmentation for maintenance"
longdesc: "a minimum fragmentation ratio to perform maintenance"

Reply:
> "Sets threshold for database maintenance, with user to authorise" ??

threshold is freepage_ratio, check is when to check and if freepage ratio is reached it asks to (or simply performs) perfrom database maintenance...
how to best and in simplest way to describe that? Needs to be universally understandable and "friendly" :)